testsuite/tests/reproducibility/cmis_on_file_system.ml

```diff
@@ -24,3 +24,5 @@
     at a given point in time *)
 type t = int
 let () = ()
+
+module M = Cmis_on_file_system_companion
```

Comment:
If I understand correctly, this PR slightly changes what the test is testing. Before we were checking that if a `.mli` is present locally but not used at all, it is not recorded at all. Now we are checking that if it is used, but only in module aliases (and we use `-no-alias-deps`), then it is not recorded differently if the `.cmi` is present on the system or if it is missing.

I would feel better with two tests to check these two different things. But then I guess @Octachron who worked on the previous reproducibility issues may have a better gut feeling of whether it is robust enough to test only the second thing.

Reply:
I considered it, but decided that this was reasonable as an extension of the previous test. It just adds more operations that should not involve reading the cmi file.

Reply:
I think it is a natural extension of the current test: we have a file that can be compiled with and without the presence of a companion cmi file and we are testing that we cannot deduce the state of the file system from the result of the compilation.

Reply:
Thanks, marking this subthread "resolved".